src/Surfnet/StepupMiddleware/CommandHandlingBundle/Tests/Identity/CommandHandler/SecondFactorRevocationTest.php

```diff
@@ -286,7 +289,14 @@ public function an_identity_can_revoke_its_own_vetted_second_factor()
                     $secondFactorId,
                     $secondFactorType,
                     $secondFactorIdentifier
-                )
+                ),
+                new VettedSecondFactorsAllRevokedEvent(
+                    $identityId,
+                    $institution,
+                    $secondFactorId,
+                    $secondFactorType,
+                    $secondFactorIdentifier
```

Comment:
Last 3 paramaters are not required right?

Reply:
Good catch, removed them